src/@chakra-ui/stories/Colors.stories.tsx

```diff
@@ -0,0 +1,185 @@
+import { type ReactNode } from "react"
+import capitalize from "lodash/capitalize"
+import { Box, Flex, HStack, Square, Stack, Text } from "@chakra-ui/react"
+import type { Meta, StoryObj } from "@storybook/react"
+
+import colors from "@/@chakra-ui/foundations/colors"
+import semanticTokens from "@/@chakra-ui/semanticTokens"
+
+const semanticTokenColors = semanticTokens["colors"]
+
+import Heading from "@/components/Heading"
+
+const meta = {
+  title: "Design System/Colors",
+  parameters: {
+    // Do not create snapshots for any stories in the file.
+    chromatic: { disableSnapshot: true },
+  },
+} satisfies Meta
+
+export default meta
+
+const primitiveColorKeys = ["gray", "blue", "orange"]
+export const Primitives: StoryObj = {
+  render: () => {
+    const primitiveColorsMap = Object.entries(colors)
+      .filter((obj) => primitiveColorKeys.includes(obj[0]))
+      .reduce<{ [tokenKey: string]: [string, string][] }>(
+        (acc, [key, value]) => {
+          const tokenMap = Object.entries(value)
+          return {
+            ...acc,
+            [key]: tokenMap,
+          }
+        },
+        {}
+      )
+
+    return (
+      <Stack direction="column" gap="16">
+        {primitiveColorKeys.map((color) => (
+          <ColorGroupWrapper key={color} color={color}>
+            <HStack>
+              {primitiveColorsMap[color].map(([tokenKey, value]) => (
+                <Stack key={`${color}${tokenKey}`} direction="column">
+                  <Box>
+                    <Square size="20" bg={`${color}.${tokenKey}`}></Square>
+                  </Box>
+                  <Stack direction="column">
+                    <Text size="sm">{value}</Text>
+                    <Text size="sm">
+                      {color}.{tokenKey}
+                    </Text>
+                  </Stack>
+                </Stack>
+              ))}
+            </HStack>
+          </ColorGroupWrapper>
+        ))}
+      </Stack>
+    )
+  },
+}
+
+const ColorGroupWrapper = ({
+  color,
+  children,
+}: {
+  color: string
+  children: ReactNode
+}) => {
+  return (
```

Comment:
<img width="1618" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/57104d26-2074-4a45-921d-f36ff773547b">

@TylerAPfledderer Getting this when I look at the primitives in Dark Mode. Otherwise this all looks great!

Reply:
@wackerow yea... they should not be view in dark mode. Only light mode. I did not create this story with that in mind.

Need to figure out how I can update the SB config to force a color mode! That's been a back-burner task with the Chakra provider

Reply:
Maybe we could just remove the `color` and `bg` to leave it under control of the parent.

Would look like this:
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/9767471e-a606-4438-827f-7d4b6ebc6456)

Reply:
I didn't think that would make sense because we don't see the orange shades in light mode.

Reply:
Maybe here I could more easily show the blues or the oranges based on color mode. I'll take a look at that.

Reply:
Not sure I followed you there @TylerAPfledderer.

IMO we should not do anything here in terms of color mode, since the purpose of this story is to list the color palette we use. The color mode should only be reflected within the semantic tokens story.

Reply:
Then I would try and disable the color mode toggle on the toolbar. We don't want to be switching modes because we just want to view everything at the same time (mimicking what it looks like in the figma file)

Reply:
Then I would try and disable the color mode toggle on the Storybook toolbar when this is previewed. We don't want to be switching modes because we just want to view everything at the same time (mimicking what it looks like in the figma file)